### PR TITLE
Golang Logger

### PR DESCRIPTION
**Pull request by mukulagarwal08**

### CODE REVIEW

**Comment by mukulagarwal08** [comment]
@ashishps1 @nikhilkumawat03
Hey Guys would love if you guys could review and let me know what can be improved. Please feel free to add other contributors for review. TIA!

File: solutions/golang/logging-framework/Logger.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
```
Comment (nikhilkumawat03) [inline solutions/golang/logging-framework/Logger.go:3]: Is this the correct path for importing the enum and interfaces?

File: solutions/golang/logging-framework/log.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ashishps1/awesome-low-level-design/logging-framework/enum"
```
Comment (nikhilkumawat03) [inline solutions/golang/logging-framework/log.go:5]: Here as well, the path for logging-framework is different.

File: solutions/golang/logging-framework/logConfig.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
```
Comment (nikhilkumawat03) [inline solutions/golang/logging-framework/logConfig.go:3]: Check the logging-framework import path

File: solutions/golang/logging-framework/main.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package main
+
+import "github.com/ashishps1/awesome-low-level-design/logging-framework/enum"
```
Comment (nikhilkumawat03) [inline solutions/golang/logging-framework/main.go:3]: Here as well check the logging-framework path

**Comment by mukulagarwal08** [comment]
Hey @nikhilkumawat03 the import statement is according to the go module defined in the go.mod. So this gives us the independence to the golang directory to run any where it is present as long as the whole golang directory is present with the go.mod. It is a working code, have tested in my local

**Comment by mukulagarwal08** [comment]
Hey @ashishps1 @nikhilkumawat03 let me know if you have any other concerns on this PR

**Pull request merged**